recipes/csm/bld.bat

```diff
@@ -0,0 +1,10 @@
+mkdir build
+cd build
+cmake ..
+cmake --build . --target ALL_BUILD --config Release
+copy Release\csmapi.lib %LIBRARY_LIB%
+mkdir %LIBRARY_INC%\csm\
+copy ..\*.h %LIBRARY_INC%\csm\
+copy Release\csmapi.dll %LIBRARY_BIN%
```

Comment:
If it works, it would probably be cleaner to do

```sh
cmake -G "NMake Makefiles" ^
   -D CMAKE_BUILD_TYPE=Release ^
   -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
    %SRC_DIR%
if errorlevel 1 exit 1

nmake
if errorlevel 1 exit 1

nmake install
if errorlevel 1 exit 1```